services/secondary-storage/server/src/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -480,7 +480,7 @@ public Answer createConfigDriveIsoForVM(HandleConfigDriveIsoCommand cmd) {
                 for (String[] item : cmd.getVmData()) {
                     String dataType = item[CONFIGDATA_DIR];
                     String fileName = item[CONFIGDATA_FILE];
-                    String content = item[CONFIGDATA_CONTENT];
+                    String content = item[CONFIGDATA_CONTENT]; // base64
```

Comment:
you can rename the variable to `base64Content`

Reply:
and remove the comment?